tests/network/hotplug.go

```diff
@@ -405,6 +405,79 @@ var _ = SIGDescribe("nic-hotunplug", func() {
 			Entry("Migration based", decorators.MigrationBasedHotplugNICs, migrationBased),
 		)
 	})
+
+	Context("a running VM", func() {
```

Comment:
Please add a test unplugging an interface from a stopped VM, similar to what hotplug has.